README.md

```diff
@@ -558,15 +558,25 @@ If you are familiar with your skills, try to create amazing works with [this](py
 
 ## How to Contribute
 
-### Submitting Issue
+### Building Overview
+
+Pyxel is built primarily using Rust, and installed via Pip. Ensure you have the *nightly* version of Rust installed with `rustup`, or `make` will generate build errors.
+
+The `make clean build` command will not install the build, so ensure you run ``pip3 install --force-reinstall `ls -rt ./dist/*.whl | tail -n 1` `` followed by `python3 -m unittest discover ./crates/pyxel-extension/tests` to install it.
```

Comment:
IMO the pip command is too complex, maybe a new makefile target is needed.
(note that I am not a project maintainer)

I would remove the mention of the tests, that’s not really useful for game devs.